src/coap_client_zephyr.c

```diff
@@ -740,6 +741,7 @@ static int golioth_connect(struct golioth_client *client)
 {
     char uri[] = CONFIG_GOLIOTH_COAP_HOST_URI;
     char *host = &uri[sizeof("coaps://") - 1];
+    char ipv6_addr[32];
```

Comment:
Is 32 enough here? How about using `OT_IP6_ADDRESS_STRING_SIZE` (40) instead?

Reply:
Good call! I used `40` as the array size in Zephyr SDK. I will change it to `OT_IP6_ADDRESS_STRING_SIZE`.